numba/np/random/_constants.py

```diff
@@ -1222,6 +1222,7 @@
 UINT16_MAX = 65535
 UINT32_MAX = 4294967295
 UINT64_MAX = 18446744073709551615
+LONG_MAX = 2147483647
```

Comment:
I think this code needs a branch, the size of a `long` depends on the platform, this is a Windows `long`, for what Numba supports on 64 bit unix systems (`LP64`), they'll have a `long` the same as `INT64_MAX`.